exo-go/src/app_config_helpers/index.go

```diff
@@ -5,10 +5,11 @@ import (
 	"io/ioutil"
 	"path"
 
+	"github.com/Originate/exosphere/exo-go/src/app_dependency_helpers"
 	"github.com/Originate/exosphere/exo-go/src/types"
 	"github.com/Originate/exosphere/exo-go/src/util"
 	"github.com/pkg/errors"
-	"github.com/segmentio/go-prompt"
+	prompt "github.com/segmentio/go-prompt"
```

Comment:
This seems like an unrelated change. Did you change this, goimports change it or something else?